README.md

```diff
@@ -505,7 +505,8 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayush Raj](https://github.com/imyoungsparda)
   - [Ayushi Vamne](https://github.com/Ayushi15-Vamne) 
   - [Aziz Prabowo](https://github.com/azizp128)
-  
+  - [Ayush Sarode](https://github.com/AyushSarode)
```

Comment:
Please add your name in line 505, after Ayush Mishra.
Let us know if you need any help with that. 🙂